src/rpc/core_rpc_server_commands_defs.h

```diff
@@ -329,11 +329,15 @@ namespace cryptonote
       crypto::public_key key;
       rct::key mask;
       bool unlocked;
+      uint64_t height;
+      crypto::hash txid;
```

Comment:
Major is for breaking compatibility, minor is for other changes.
In this case, I think the best might be to increment minor, and only display the graph if the daemon is at least this version. This will allow a newer wallet (with your code) to still work with an older daemon, which would not be sending height and txid in its response. This is deemed good since many people using the GUI are likely to use remote daemons, and RPC changes a lot.